src/verifiers/issuerIdentity/dnsDid/dnsDidProof.ts

```diff
@@ -96,20 +94,32 @@ const verifyV2 = async (document: v2.WrappedDocument): Promise<VerificationFragm
     return verifyIssuerDnsDid({ key, location });
   });
   const verificationStatus = await Promise.all(deferredVerificationStatus);
-  const overallStatus =
-    verificationStatus.some((status) => status.status === "VALID") &&
-    verificationStatus.every((status) => status.status === "VALID")
-      ? "VALID"
-      : "INVALID";
+
+  if (verificationStatus.length > 0 && ValidDnsDidVerificationStatusArray.guard(verificationStatus)) {
```

Comment:
same as above, might be better with length guard in runtype?

Reply:
done